src/lib/createStardustComponent.tsx

```diff
@@ -0,0 +1,36 @@
+import createComponentInternal from './createComponent'
+import * as React from 'react'
+import * as _ from 'lodash'
+import { ComponentSlotClasses } from '../themes/types'
+
+export interface RenderStardustResultConfig {
```

Comment:
no `styles`?

Reply:
We are providing the classes to user, which they can safely use in the className props. The styles should be just input for the createComponent that will create the classes for the user.